astronomer/airflow/version_check/update_checks.py

```diff
@@ -457,11 +462,34 @@ def available_eol(self) -> dict[str, Any] | None:
         )
         return self.get_eol_notice(current_version)
 
+    def available_yanked(self) -> dict[str, Any] | None:
+        """Check if the current version of Astronomer Runtime is yanked."""
+        from .models import AstronomerAvailableVersion
+
+        session = get_state(app=current_app).db.session
+        runtime_version = version.parse(get_runtime_version())
+        current_version = (
+            session.query(AstronomerAvailableVersion)
+            .filter(AstronomerAvailableVersion.version == str(runtime_version))
+            .one_or_none()
+        )
+
+        if current_version and current_version.yanked:
+            return (
```

Comment:
@cmarteepants can you sign off on the message wording here?

Reply:
How about this?

> "Warning: This version of Astronomer Runtime, {} has been yanked. We strongly recommend upgrading to a more recent supported version"
...

Reply:
Changed it.